tests/test_utils.py

```diff
@@ -14,3 +15,39 @@ def test_read_files(self):
             self.assertIn('proto_signature', line)
             self.assertIn('signature', line)
             self.assertIn('uuid', line)
+
+
+class StackTraceProcessorTest(unittest.TestCase):
+    raw_traces = ['{"proto_signature": "a | CAPITAL_LETTERS | c", "uuid": "1", "signature": "c"}',
+                  '{"proto_signature": "a | b | d", "uuid": "2", "signature": "d"}',
+                  '{"proto_signature": "a | x | d", "uuid": "3", "signature": "d"}',
+                  '{"proto_signature": "a | b | e", "uuid": "4", "signature": "same"}',
+                  '{"proto_signature": "with | name@0xAddr | drop", "uuid": "5", "signature": "drop"}',
+                  '{"proto_signature": "with | xul.dll@", "uuid": "6", "signature": "ignored"}',
+                  '{"proto_signature": "a | b | e", "uuid": "7", "signature": "same"}']
+
+    expected_traces = [(['a', 'capital_letters', 'c'], 'c', '1'),
+                       (['a', 'b', 'd'], 'd', '2'),
+                       (['a', 'x', 'd'], 'd', '3'),
+                       (['a', 'b', 'e'], 'same', '4'),
+                       (['with', 'name@0x', 'drop'], 'drop', '5')]
+
+    def test_clean_func(self):
+        funcs = [('js::jit::MakeMRegExpHoistable ', 'js::jit::makemregexphoistable'),
+                 (' AppKit@0x7be82f ', 'appkit@0x'),
```

Comment:
Yes, in this case yes :)

I like this format, but feel free to leave it as it is:
```
funcs = [
    ('js::jit::MakeMRegExpHoistable ', 'js::jit::makemregexphoistable'),
    (' AppKit@0x7be82f ', 'appkit@0x'),
    ...
]
```